suite-native/module-send/src/screens/SendOutputsReviewScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import { useDispatch } from 'react-redux';
+
+import { isFulfilled } from '@reduxjs/toolkit';
+
+import {
+    Screen,
+    ScreenSubHeader,
+    SendStackParamList,
+    SendStackRoutes,
+    StackProps,
+} from '@suite-native/navigation';
+import { IconButton, VStack } from '@suite-native/atoms';
+import { cancelSignSendFormTransactionThunk } from '@suite-common/wallet-core';
+import { useTranslate } from '@suite-native/intl';
+import { useNativeStyles } from '@trezor/styles';
+
+import { ReviewOutputItemList } from '../components/ReviewOutputItemList';
+import { OutputsReviewFooter } from '../components/OutputsReviewFooter';
+import { SignSuccessMessage } from '../components/SignSuccessMessage';
+import { cleanupSendFormThunk } from '../sendFormThunks';
+
+export const SendOutputsReviewScreen = ({
+    route,
+    navigation,
+}: StackProps<SendStackParamList, SendStackRoutes.SendOutputsReview>) => {
+    const { accountKey } = route.params;
+    const { utils } = useNativeStyles();
+    const { translate } = useTranslate();
+    const dispatch = useDispatch();
+
+    const handleGoBack = async () => {
```

Comment:
is it okay, that hardware back pressed does not trigger this?

Reply:
navigation revisited in fixup 35fb89870f